airflow/contrib/operators/s3_to_gcs_transfer_operator.py

```diff
@@ -45,15 +45,14 @@ class S3ToGoogleCloudStorageTransferOperator(BaseOperator):
         For this to work, the service account making the request must have
         domain-wide delegation enabled.
     :type delegate_to: str
+    :param description: Optional transfer service job description
+    :type description: str
     :param object_conditions: Transfer service object conditions; see
         https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec
     :type object_conditions: dict
     :param transfer_options: Transfer service transfer options; see
         https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec
     :type transfer_options: dict
-    :param job_kwargs: Additional transfer job options; see
```

Comment:
We do have things like Schedule: https://cloud.google.com/storage-transfer/docs/reference/rest/v1/transferJobs#Schedule that can go in `job_kwargs`, what do you think?

Reply:
The `GCPTransferServiceHook` already sets the schedule so that the job runs right away and only once. Then the hook waits for the job to complete. If we want to allow custom schedules, I think we'd want to make waiting optional--it wouldn't make sense to schedule a job a day in the future, then wait for it to finish. I'll revise and add those options later today.